client/battle/CreatureAnimation.cpp

```diff
@@ -115,12 +115,12 @@ float AnimationControls::getCreatureAnimationSpeed(const CCreature * creature, c
 
 float AnimationControls::getProjectileSpeed()
 {
-	return static_cast<float>(settings["battle"]["animationSpeed"].Float() * 100);

```

Comment:
multiplier of projectile speed compared to multiplier of catapult speed has different ratio now, is it intended?